core/arch/arm/plat-d02/main.c

```diff
@@ -63,34 +65,14 @@ static void main_fiq(void)
 	panic();
 }
 
-static vaddr_t console_base(void)
+void console_init(void)
```

Comment:
This function depends on being first with mmu disabled and then once again with mmu enabled. That seems a bit counterintuative. What @etienne-lms suggests above with `console_early_init()` would make it more clear.